### ISSUE TITLE
Card processing error still displays when atempting to validate again

### ISSUE THREAD

**Issue by bertso** [citymenu/clickandeat#204]
Sorry about the title, I need 50000 characters to express myself. When doing a card payment I entered the wrong details and it took me back to the beginning of the card processing and show me the error message, which is correct, but then when I entered the correct details and try again it keeps on displaying the error in the subsequent windows, although everything is working fine. I guess its just a question of hiding\removing\clearing the div\span that is displaying the error

**Comment by mishimaltd** [comment]
Yeah, oops :-)

On 23 Oct 2012, at 08:22, bertso notifications@github.com wrote:

Sorry about the title, I need 50000 characters to express myself. When
doing a card payment I entered the wrong details and it took me back to the
beginning of the card processing and show me the error message, which is
correct, but then when I entered the correct details and try again it keeps
on displaying the error in the subsequent windows, although everything is
working fine. I guess its just a question of hiding\removing\clearing the
div\span that is displaying the error

—
Reply to this email directly or view it on
GitHubhttps://github.com/mishimaltd/clickandeat/issues/204.
